lib/grinder_sdk.dart

```diff
@@ -44,7 +44,7 @@ class Dart {
   /// Returns the stdout.
   static String run(String script,
       {List<String> arguments : const [], bool quiet: false,
-       String packageRoot, String workingDirectory, int vmNewGenHeapMB,
```

Comment:
I would love to deprecate for now instead of remove. The more we can easy people along API changes the better.